sponsorship_compassion/models/res_partner.py

```diff
@@ -121,8 +121,16 @@ def _set_count_items(self):
 
     @api.multi
     def _compute_children(self):
+        church = None
+        total = 0
         for partner in self:
-            partner.number_children = len(partner.sponsored_child_ids)
+            if partner.is_church:

```

Comment:
No, this should as well be in compassion-switzerland. Here, the field `is_church` doesn't exist.